storage/src/adb/any/fixed/mod.rs

```diff
@@ -820,6 +696,25 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Array, H: CHasher, T: Translato
     }
 }
 
+/// Return true if the given sequence of `ops` were applied starting at location `start_loc` in
+/// the log with the provided root.
+pub fn verify_proof<H, K, V>(
```

Comment:
Is this needed still? (can we just call the one in verify.rs instead?)

Reply:
Removed